fabric-events-lifecycle-v0/src/main/java/net/fabricmc/fabric/api/event/server/ServerSaveCallback.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.event.server;
+
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.world.level.LevelProperties;
+
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+
+/**
+ * A callback for the server's saving.
```

Comment:
> saving

Could use some elaboration on *what* is getting saved... level.dat, some chunks, all the chunks, player files, or something else?

Reply:
server's saving is the server getting saved, including level.dat, all chunks, player files, etc. Partial chunk saving is not server saving.